### PR TITLE
Add enrolment data to dummy data script & create reusable methods

### PR DESCRIPTION
**Pull request by brittanylau**
chonky PR but mostly due to renaming files & tests!

### Notion ticket link

[Dummy data for sessions, enrolments, and classes](https://www.notion.so/uwblueprintexecs/9cb454262ed84c6299c0682c23f92ec5?v=1076bd30711a4a01838becee05f8b5a3&p=d338942b1f634878bd3cc8d6778809c8)

### Implementation description

- created test util functions to create a bunch of objects
- add tests...for the test utils
- added sessions, enrolments, and classes to the dummy data script

### Steps to test

1. python manage.py test
2. python manage.py load_initial_data --families X --sessions X --classes_per_session X

### What should reviewers focus on?

- whether objects are being made correctly (mostly just running the test script and checking the results)

### Checklist

- [x] My PR name is descriptive and in imperative tense
- [x] I have run the linter
- [x] I have requested a review from the PL, and/or other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

**Comment by Tayef-Shah** [comment]
All the data seemed to generate correctly, but I noticed in Django Admin, there's no session column under the classes. Not a big deal since I knew I had 3 sessions with 3 classes in each, so 9 in total but just wanted to point it out.

Screen Shot 2021-04-27 at 9 33 31 PM

File: enrolments/management/commands/load_initial_data.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+                for session in sessions:
+                    classes.extend(
+                        enrolment_utils.create_test_classes(
+                            session=session,
+                            num_classes=num_classes_per_session,
+                        )
+                    )
+
+                num_classes = num_sessions * num_classes_per_session
+                num_families_per_class = -(-num_families // num_classes)
```
Comment (Tayef-Shah) [inline enrolments/management/commands/load_initial_data.py]: woah this is a cool notation. I'm gonna assume u know what it means lol, cuz the minus signs kinda lost me

File: enrolments/management/commands/load_initial_data.py [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+                        f"Successfully created {Field.objects.all().count()} fields, "
+                        f"{Family.objects.all().count()} families, and "
+                        f"{len(Student.objects.all())} students"
+                    )
+                )
+
+        except:
+            if verbose:
+                self.stdout.write(self.style.ERROR(f"Something went wrong"))
+            raise
```
Comment (Tayef-Shah) [inline enrolments/management/commands/load_initial_data.py:114]: should we have a new line here after this line?

File: enrolments/tests/utils/utils.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    sessions = []
+    for i in range(num_sessions):
+        sessions.append(
+            Session(
+                season=Session.SEASON_CHOICES[i % len(Session.SEASON_CHOICES)][1],
+                year=2020 - i // len(Session.SEASON_CHOICES),
+            )
+        )
+
+    return Session.objects.bulk_create(sessions)
```
Comment (Tayef-Shah) [inline enrolments/tests/utils/utils.py:19]: Oh I wish I knew about `bulk_create` earlier.

File: registration/tests/utils/utils.py [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+        with_fields=with_fields,
+        num_children=num_children,
+    )
+    create_test_guests(
+        family=family,
+        last_name=last_name,
+        with_fields=with_fields,
+        num_guests=num_guests,
+    )
+    return family
```
Comment (Tayef-Shah) [inline registration/tests/utils/utils.py:153]: nit: should we have a new line?

**Review by Tayef-Shah — APPROVED** [review]
LGTM! This is very pog, thanks @brittanylau!

**Comment by brittanylau** [comment]
> All the data seemed to generate correctly, but I noticed in Django Admin, there's no session column under the classes. Not a big deal since I knew I had 3 sessions with 3 classes in each, so 9 in total but just wanted to point it out.

good call! added sessions to the class admin 😄 thanks for reviewing!

File: enrolments/management/commands/load_initial_data.py (hunk 2 above)
Comment (brittanylau) [reply enrolments/management/commands/load_initial_data.py:114]: there is! i think github just doesn't show it - if a newline is missing it usually has a lil ⛔ symbol

File: enrolments/tests/utils/utils.py (hunk 3 above)
Comment (brittanylau) [reply enrolments/tests/utils/utils.py:19]: it's pretty neat! saves transactions if you're creating lots of objs

File: registration/tests/utils/utils.py (hunk 4 above)
Comment (brittanylau) [reply registration/tests/utils/utils.py:153]: it does! github hides em 🙈

File: enrolments/management/commands/load_initial_data.py (hunk 1 above)
Comment (brittanylau) [reply enrolments/management/commands/load_initial_data.py]: this is ceiling division! python doesn't have a built in operator, so we [negate floor division](https://stackoverflow.com/a/17511341) instead

added a comment to clarify what's goin on!

**Pull request merged**